static/js/publisher/release/components/releasesConfirmDetails/progressiveRow.js

```diff
@@ -54,15 +65,21 @@ const ProgressiveRow = ({
   } else {
     progress = (
       <Fragment>
-        <InteractiveProgressiveBar
-          percentage={startingPercentage}
-          onChange={updatePercentage}
-          targetPercentage={targetPercentage}
-          minPercentage={1}
-          singleDirection={type === progressiveTypes.UPDATE ? 1 : 0}
-        />
+        {globalPercentage &&
```

Comment:
There is a lot of conditionals in this JSX making it hard to deduce what is meant to show up when.

`globalPercentage && !updateGlobalPercentage` ... `(!globalPercentage || updateGlobalPercentage)`.

Seems like there are 2 possible states - show progress bar or show interactive progress bar. Would be nice if JSX would show this simplicity and the conditional logic would be moved to code.